tools/cli/app.go

```diff
@@ -247,6 +247,9 @@ func NewCliApp(cf ClientFactory, opts ...CLIAppOptions) *cli.App {
 		printMessage(output, "command not found: "+command)
 	}
 
+	// DisableSliceFlagSeparator is set to true to allow having comma in JSON values for slice flags
+	app.DisableSliceFlagSeparator = true
+
```

Comment:
I wonder if things like start workflow execution is also broken because of this? It takes a JSON input.